parsing/ast_helper.ml

```diff
@@ -19,9 +19,11 @@ open Asttypes
 open Parsetree
 open Docstrings
 
-type lid = Longident.t loc
-type str = string loc
+type 'a with_loc = 'a Location.loc
```

Comment:
Does this suggest that `'a Location.loc` would be better named `'a Location.with_loc`? (but changing that now may be not worth the hassle..)

Reply:
Yes?